tests/translate/translate_updatedzd.py

```diff
@@ -25,18 +30,43 @@ def __init__(self, grid):
             self.out_vars[v] = self.in_vars["data_vars"][v]
         self.out_vars["wsd"]["kstart"] = grid.npz
         self.out_vars["wsd"]["kend"] = None
+        self.updatedzd: Optional[UpdateHeightOnDGrid] = None
 
     def compute(self, inputs):
         self.make_storage_data_input_vars(inputs)
-        updatedzd = fv3core.stencils.updatedzd.UpdateDeltaZOnDGrid(
+        self.updatedzd = fv3core.stencils.updatedzd.UpdateHeightOnDGrid(
             self.grid,
+            spec.namelist,
             inputs.pop("dp0"),
             d_sw.get_column_namelist(spec.namelist, self.grid.npz),
             d_sw.k_bounds(),
         )
         inputs["x_area_flux"] = inputs.pop("xfx")
         inputs["y_area_flux"] = inputs.pop("yfx")
-        updatedzd(**inputs)
+        inputs["surface_height"] = inputs.pop("zs")
+        inputs["height"] = inputs.pop("zh")
+        inputs["courant_number_x"] = inputs.pop("crx")
+        inputs["courant_number_y"] = inputs.pop("cry")
```

Comment:
to use different names in the model than what was defined for serialization, we set the 'serialname' attribute to the in_vars dictionary. e.g. 
instead of 
"zs": {},
use
"surface_height": {"serialname": "zs"},